eng/common/scripts/add-pullrequest-reviewers.ps1

```diff
@@ -18,6 +18,31 @@ param(
     $AuthToken
 )
 
+$errorOccurred = $false
+
+function AddMembers($memberName, $additionSet) {
+  $headers = @{
+    Authorization = "bearer $AuthToken"
+  }
+  $uri = "https://api.github.com/repos/$RepoOwner/$RepoName/pulls/$PRNumber/requested_reviewers"
+
+  foreach ($id in $additionSet) {
+    try {
+      $postResp = @{}
+      $postResp[$memberName] = @($id)
+      $postResp = $postResp | ConvertTo-Json
+
+      Write-Host $postResp
+      $resp = Invoke-RestMethod -Method Post -Headers $headers -Body $postResp -Uri $uri -MaximumRetryCount 3
+      $resp | Write-Verbose
+    }
+    catch {
+      Write-Host "Error attempting to add $user `n$_"
+      $errorOccurred = $true
```

Comment:
If you are trying to set the outer scope I think you need `$script:errorOccurerd`

Reply:
You're absolutely right. I made that adjustment, then after staring at it for a couple minutes, I just simplified it further. No more scoping issues.